src/runtime/arch/s390x-options.mk

```diff
@@ -13,8 +13,8 @@ CPUFEATURES :=
 QEMUCMD := qemu-system-s390x
 
 # See https://github.com/kata-containers/osbuilder/issues/217
-FEDORA_LIKE = $(shell grep -E "\<fedora\>" /etc/os-release 2> /dev/null)
-ifneq (,$(FEDORA_LIKE))
+RPM_BASED = $(shell which rpm)
```

Comment:
I'm not sure if this is going to be 100% reliable: some of my Ubuntu system have `rpm` installed ;)

In fact, it would be useful if `/etc/os-release` specified something like:

```
PACKAGE_MANAGER=[apt|yum|dnf|zypper|snap]
PACKAGE_FORMAT=[dpkg|rpm|snap]
```

Reply:
Updated.